STEERING-COMMITTEE.md

```diff
@@ -100,19 +97,25 @@ first name):
 \* Open seats have been allocated to the organization per the committee rules,
   but have not been assigned to a member of the organization.
 
-### Allocation of seats
+### Composition
+
+As of 2020, the three organizations with the largest demonstrated 
+commitment to the project are Google, IBM/RedHat, and VMWare. For
+2021, a 5 person steering committee should be competent to
+execute its responsibilities, with three seats for Google, one seat
+for IBM/RedHat, and one seat for VMWare.
 
-Seats on the steering committee are allocated based upon contribution to the
-project by an organization. No final decision has been made on the exact
-formula.
+### Nomination of representatives
 
-As the project continues to grow, we expect to add additional seats to the
-committee, ensuring that those contributing to the project are properly
-represented.
+Seats on the steering committee are allocated based upon lifetime
+contributions to the project by an organization. The intention is
+to ensure that those organizations are committed to the ongoing
+maintenance burden that adding to core implies, and express their
```

Comment:
this is really a downer for any independent contributor. You are thriving for a "vibrant" project but put the bar for membership in the steering at "corporation committed to ongoing maintenance burden", any freelancer/independent, small business, lone ranger etc is totally dis-incentivized to participate and engage fully. I understand technical leadership is in the TOC but some independent may want to engage and help steer as well. Saying "well you are not big enough, you don't have enough breadth to support maintenance" is an invitation for people to not engage.

We are now 2 years into this project and let's face it the contributions from startups and end-user companies are very very small, this is not a vibrant project.